### ISSUE TITLE
Allow setting to mute backintime

### ISSUE THREAD

**Issue by Germar** [bit-team/backintime#427]
I have multiple profiles for backing up my laptop to different media. When I start the GUI version of backintime, I am often greeted with a dialogue box stating that it "Can't find snapshots folder", and a beep.

The dialogue box is annoying enough, as it occurs so often, but I can understand the logic behind it. OTOH, it'd be nice if you could turn off the sound in backintime, as it's not really necessary. In fact, I can't think of many other programs at all that use sounds to alert the user when something is wrong.

Backintime 1.0.10
Kubuntu 13.04

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2013-09-13T01:03:06Z
 * owner: sparhawkthesecond
 * the launchpad url was https://bugs.launchpad.net/bugs/1224745

**Comment by Germar** [comment]
(by germar)
Backintime by it self doesn't play any sound. If you have sound on error messages this will come from your system settings. You can use attached snippet to test.

**Comment by Germar** [comment]
(by sparhawkthesecond)
Ah, thanks.

I couldn't actually get your script to play a sound, but it inspired me to some other troubleshooting. I use KDE, and the alert sound in backintime only occurs when I launch backintime with `kdesudo backintime` (which is how I always use it). When I launch it as a normal user there is no alert sound. Hence, I needed to mute system sounds for the superuser with `kdesudo systemsettings` (it was already muted for the user).

(FWIW I couldn't get the script to play a sound even when I enabled sounds for the user. Nor could I get it to play a sound with `kdesudo python /path/to/errmsg.py`.)

Sorry for the erroneous bug report, and thanks for your time.
